src/components/NewNursery/index.tsx

```diff
@@ -162,18 +138,10 @@ export default function NurseryView(): JSX.Element {
             </Grid>
             {timeZoneFeatureEnabled && (
               <Grid item xs={gridSize()}>
-                <TimeZoneSelector
-                  selectedTimeZone={record?.timeZone || defaultTimeZone.id}
-                  onTimeZoneSelected={onChangeTimeZone}
-                  label={strings.TIME_ZONE}
-                  disabled={orgTZChecked}
-                />
-                <Checkbox
-                  label={strings.USE_ORG_TZ}
-                  onChange={(value) => onOrgTimeZoneChecked(value)}
-                  id='orgTZ'
-                  name='orgTZ'
-                  value={orgTZChecked}
+                <TimeZoneSelectorWithCheckbox
```

Comment:
Call it `LocationTimeZoneSelector` since it is use for locations (seed bank, nursery, planting site)